import createError from "http-errors"
const handleNotFound = (result) => {
    if (!result) {
        throw createError(404, 'trooper n√£o encontrado!')
    }
    return result
}

export {handleNotFound}